modules/aerospike/aerospike_test.go

```diff
@@ -0,0 +1,64 @@
+package aerospike_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	as "github.com/testcontainers/testcontainers-go/modules/aerospike"
```

Comment:
suggestion: let's be consistent with other modules
```suggestion
	tcaerospike "github.com/testcontainers/testcontainers-go/modules/aerospike"
```